modules/@apostrophecms/area/ui/apos/components/AposAreaExpandedMenu.vue

```diff
@@ -28,9 +32,28 @@
                 }`
               ]"
             >
+              <button
+                v-for="(item, itemIndex) in group.operations"
+                v-if="group.type === 'operations'"
```

Comment:
don't these cancel each other?

Reply:
They agree with each other. *If* the group type is operations, *then* we iterate over the operations.